### ISSUE TITLE
More options: preselect a currency , and multiselect several currencies

### ISSUE THREAD

**Issue by giorgio79** [Daniel-Ioannou/flutter_currency_picker#9]
Would be nice to have a preselect array option, so one or more currencies can be preselected.
I am working on an app that has a currency selector and do some geolocation magic, and preselect the currency based on user geolocationn https://play.google.com/store/apps/details?id=com.cashtic

Also, the capability to multiselect would be great too.

**Comment by giorgio79** [comment]
If anyone needs this functionality here is the code for getting user currency by location. It needs the geolocator and Google Maps api for reverse geocoding

```
      userGeolocatorPlacemark = await placemarkFromCoordinates(
          globals.geolocatorPosition.latitude,
          globals.geolocatorPosition.longitude);

      //get currency code for country
      //https://stackoverflow.com/questions/58766133/how-to-get-currency-and-currency-symbol-in-flutter-by-country-code
      userCurrencyByLocationCountry = CurrencyPickerUtils.getCountryByIsoCode(
              userGeolocatorPlacemark[0].isoCountryCode)
          .currencyCode;
```

**Comment by Daniel-Ioannou** [comment]
@giorgio79 I'm not sure what you mean preselected currencies, package dose not keep the selected currency, it only return the selected currency through `onSelect` callback. 

If you want to show one or more currencies at the top of the list you can use `favorite` parameter:
```Dart
   showCurrencyPicker(
     context: context,
     onSelect: (Currency currency) {
       print('Select currency: ${currency.name}');
     },
     favorite: <String> ['EUR', 'GBP', 'USD'],
   );
```

**Comment by giorgio79** [comment]
Yeah, so if someone selects values and then the app saves it to Firebase or some storage, the next time the user edits the selected currencies, the saved values could be preloaded...

**Comment by voivoed** [comment]
@giorgio79 the way I handled this on my app was to put the previously selected currency as a favorite, so it shows at the top of the list in a separate box right under the search box. The user can just tap it again right away.

**Comment by MLZ91** [comment]
A preselected country is possible right now but i have to use an ugly workaround. It would be nice if you could make currencies.dart available through a getter  so we could set a new/initial Currency easily.
